### DISCUSSION TITLE
Best way to get current environment

### DISCUSSION THREAD

**Discussion by jsaxdev** [abuzuhri/Amazon-SP-API-CSharp#93] [Q&A] [answered]
What's the best way to get current environment (production, sandbox) in code ?
@abuzuhri said that you can get it from settings but I do not find these settings.
You can get from amazon credentials but you need to have it initialized.
There is EnvironmentManager in ApiUrls file but nothing references it.

**Comment by abuzuhri** [comment, marked as answer]
You can git it from  `AmazonCredential.Environment` 
you can access this from service
